File: java/src/main/java/itp/itp1_4/ITP1_4_C.java
package itp.itp1_4;

import java.io.PrintWriter;
import java.util.HashMap;
import java.util.Map;
import java.util.Scanner;

public class ITP1_4_C {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    PrintWriter pw = new PrintWriter(System.out);
    while (true) {
        int a = sc.nextInt();
        String op = sc.next();
        int b = sc.nextInt();
        if ("?".equals(op)) break;
        Map<String, Integer> result = new HashMap<>();
        result.put("+", a + b);
        result.put("-", a - b);
        result.put("*", a * b);
        result.put("/", b != 0 ? a / b : 0);
        pw.append(String.valueOf(result.get(op)));
        pw.append("\n");
    }
    pw.flush();
  }
}

File: kotlin/src/itp/itp1_4/ITP1_4_C.kt
package itp.itp1_4

import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    while (true) {
        val a = sc.nextInt()
        val op = sc.next()
        val b = sc.nextInt()
        if (op == "?") break
        val ans = when (op) {
            "+" -> a+b
            "-" -> a-b
            "*" -> a*b
            "/" -> a/b
            else -> throw IllegalStateException()
        }
        println(ans)
    }
}